web-common/static/index.html

```diff
@@ -4,16 +4,24 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
 
-        <title>Meadow</title>
```

Comment:
don't we want a title?

Reply:
It's set in the webpack plugin. That makes it easier to use one file configured for two apps.